engine/src/distribution/primitives.cpp

```diff
@@ -29,13 +30,18 @@ typedef blazingdb::manager::Context Context;
 typedef blazingdb::transport::Node Node;
 
 
-
 std::unique_ptr<BlazingTable> generatePartitionPlans(
-				cudf::size_type number_partitions, const std::vector<BlazingTableView> & samples,
-				const std::vector<cudf::order> & sortOrderTypes) {
+	cudf::size_type number_partitions,
```

Comment:
why does this takea  table instead of a view? a table has to be moved a view cna be copied

Reply:
The function that calls this one used to take views, but needed to now take tables. So we updated this one as well, since most of our code now operates on tables, not views